docs/agility-resources/facilitation-notes.md

```diff
@@ -0,0 +1,46 @@
+# Facilitation
+
+## Meeting Agendas
+
+### Inceptions
+
+1. Intro & Ground Rules (5 min)
+2. Project Scope and Motivation (10-15 min)
+3. Proposed Solution
+
+_break_
+
+5. Brainstorming session - areas of investment to address the proposed solution (45 min)
+
+_5 min break, Facilitators do a grouping exercise_
+
+6. Risks & Mitigations (1 hour)
+
+_break (usually lunch time)_
+
+7. Epic definitions, initial story writing (1 hour)
+8. Retro (30 min)
+
+_go home_
+
+### Ground Rules
+
+**Team Health Assessments**
```

Comment:
@menicosia - did you mean to include this line?

Reply:
(line 28)

Reply:
I did. I can update to make it more clear - that's the standard agenda (from Joey's recollection) for team health assessments.

I put it this way because maybe there are different ground rules for different types of meetings, like Inceptions are maybe a bit different...?